playbooks/bootstrap.yml

```diff
@@ -1,6 +1,35 @@
 ---
-- import_playbook: "./bootstrap_tower.yml"
-- import_playbook: "./bootstrap_integreatly.yml"
-- import_playbook: "./bootstrap_cluster_create.yml"
-- import_playbook: "./bootstrap_cluster_teardown.yml"
-- import_playbook: "./bootstrap_validation.yml"
\ No newline at end of file
+- hosts: localhost
+  gather_facts: no
+  tasks:
+    - debug:
+        msg: "ERROR: No branch/tag of the configuration repo defined defaulting to master"
+      when: integreatly_version is not defined
```

Comment:
I would nearly go as far as having a (y/n) prompt if `integreatly_version` is not defined. For our e2e tests we can just override it to master e.g. `-e integreatly_version=master`